rasa/train.py

```diff
@@ -32,7 +32,11 @@ def train(
     loop: Optional[asyncio.AbstractEventLoop] = None,
 ) -> Optional[Text]:
     if loop is None:
-        loop = asyncio.get_event_loop()
+        try:
+            loop = asyncio.get_event_loop()
+        except RuntimeError:
```

Comment:
This can only happen when running from inside the executor, right?

Reply:
yep, should I add a comment?

Reply:
added a comment